### DISCUSSION TITLE
new messages not shown

### DISCUSSION THREAD

**Discussion by lsloan** [community/community#3189] [Discussions] [answered]
I had posted a [question](https://github.com/Homebrew/discussions/discussions/555) and later added some comments for clarification.  I then left the page open in my browser to see replies as they arrived.  After a day and no new comments appeared, I asked the other user in the conversation if they had anything to add.  They replied that they had added a comment, but I didn't see it.

I had to reload the page, then the other comments were visible.  That's unusual for GitHub.  I'm used to various features (issues, pull requests, code reviews) updating dynamically as new information is added.  (Or at the very least, then giving an indication that the page needs to be reloaded.)

**Comment by evi-liu** [comment, marked as answer]
Thanks for the feedback. We'll look into live updating to make sure there's at least parity with our other GitHub experiences.

**Reply by 65156** [reply]
Hi @evi-liu we have noticed on our GHE server after a recent upgrade that our comments are now no longer updating live, and we need to now manually refresh to get updates, can you advise me what setting needs to be tweaked for this?
